bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+* Title accurately describes the content
+* Proposal is of general interest and/or pertains to more than one Bitcoin project/implementation
+* Document is properly formatted
+* Licensing terms are acceptable
+* Motivation, Rationale, and Backwards Compatibility have been addressed (where applicable)
+* The defined Layer header must be correctly assigned for the given specification
+* The BIP is ready: it is complete, comprehensible, and technically feasible
+
+Editors do NOT evaluate whether the proposal is likely to be adopted.
+
+A BIP editor will:
+
+* Assign a BIP number in the pull request
+* Merge the pull request when it is ready
+* List the BIP in the [README](README.mediawiki)
+
+The BIP editors are intended to fulfill administrative and editorial responsibilities. The BIP editors monitor BIP
+changes, and update BIP headers as appropriate.
+
+BIP editors may also, at their option, unilaterally make and merge strictly-editorial changes to BIPs, such as
+correcting misspellings, fixing broken links, etc. as long as they do not change the meaning or conflict with the
+original intent of the author.
+
+## Changes from BIP-2
+
+- Refer to the proponent of a BIP as "Champion"
+- Status field is no longer modeled around the workflow of consensus changes. The only remaining status are DRAFT,
+  NOMINATED, ACTIVE, and ABANDONED. The statuses Deferred, Final, Obsolete, Proposed, Rejected, Replaced, and Withdrawn
+  are sunset.
+- BIPs can no longer be rejected due to inactivity, but BIPs that get no adoption whose champion has become unreachable can be moved to Abandoned
+- Judgment calls required from BIP Editors are mostly reassigned either to the audience or the BIP champion
+- Tracking of adoption, acceptance, and community consensus is out of scope for the BIPs repository
+- Layer header is no longer restricted to Standards Track
+- Auxiliary files are only permitted in the corresponding BIP’s subdirectory as nobody used the alternative of labeling
+  them with the BIP number
+- The Comments-URI and Comment-Summary headers are dropped from the preamble, Comments as an aspect of the process are
+  discontinued[^comments]
+- The Discussions-To header is dropped, as it has never been used
+- List of acceptable licenses was reduced to the ones previously used
+- "Other Implementations" sections are discouraged
+
+## Copyright
+
+This BIP is licensed under the [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/) license.
+
+## Related Specifications
+
+- [BIP 1: BIP Purpose and Guidelines](bip-0001.mediawiki)
+- [BIP 2: BIP Process, revised](BIP-0002.mediawiki)
+- [BIP 123: BIP Classification](BIP-0123.mediawiki)
+* [RFC 7282: On Consensus and Humming in the IETF](https://tools.ietf.org/html/rfc7282)
+
+## Rationale
+
+[^astroturfing]: **What does it mean to be focused on Bitcoin the currency?**  
+    Proposals to astroturf on the Bitcoin network to store data, bootstrap their own consensus mechanism, or facilitate
```

Comment:
Or if you have a definition of what it means to "follow Bitcoin philosophy" that could fit this document, I’d also be open to define it, so it can be evaluated in this context.

Reply:
I don't think the word "astroturf" is even needed here?

Would "bootstrap their own consensus methods" preclude publishing a BIP for something like [spacechains](https://github.com/RubenSomsen/spacechains) -- that is a sidechain whose native currency (in which fees are paid) is pegged to BTC, where the chain tip is determined by bumping a utxo on bitcoin (ie sidechain miners compete by fee-bumping a bitcoin transaction, rather than doing direct proof-of-work)? That doesn't really seem optimal to me; but I suppose it could be revisited if/when such a proposal ever becomes practical/supported, so whatever.